modules/playlists/src/main/java/org/opencastproject/playlists/persistence/PlaylistDatabaseService.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ *
+ * The Apereo Foundation licenses this file to you under the Educational
+ * Community License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License
+ * at:
+ *
+ *   http://opensource.org/licenses/ecl2.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+package org.opencastproject.playlists.persistence;
+
+import org.opencastproject.playlists.Playlist;
+import org.opencastproject.util.NotFoundException;
+import org.opencastproject.util.requests.SortCriterion;
+
+import java.util.List;
+
+/**
+ * API that defines persistent storage of playlists
+ */
+public interface PlaylistDatabaseService {
+
+  /**
+   * Gets a single playlist by its identifier.
+   * @param playlistId the playlist identifier
+   * @return the {@link Playlist} with the given identifier
+   * @throws NotFoundException if there is no playlist  with this identifier
+   * @throws PlaylistDatabaseException if there is a problem communicating with the underlying data store
+   */
+  Playlist getPlaylist(String playlistId) throws NotFoundException, PlaylistDatabaseException;
+
+  /**
+   * Gets a single playlist by its identifier.
+   * @param playlistId the playlist identifier
+   * @param orgId the organisation identifier
+   * @return the {@link Playlist} with the given identifier
+   * @throws NotFoundException if there is no playlist  with this identifier
+   * @throws PlaylistDatabaseException if there is a problem communicating with the underlying data store
+   */
+  Playlist getPlaylist(String playlistId, String orgId) throws NotFoundException, PlaylistDatabaseException;
```

Comment:
This method seems out of place since it's the *only* one that lets you specify which org to use.  It's also only in use in the impl of the database, where is uses the current security environment's org anyway :D

Reply:
What do you mean by "only one"? There are other methods in that interface that let you specify organization id, right?

And yes, currently this function is only used in one place. I thought it might be nice to have it available in the future if someone needs it for ... something.

Reply:
Yeah I must have had an off day and missed the others.  Nevermind :)